pages/media-release-form.tsx

```diff
@@ -0,0 +1,109 @@
+                    avenues. I also authorize any media material created by
+                    myself within the Social Diversity for Children Foundation.
+                </Text>
+                <Text pb="5px" align="left" mt="30px">
+                    I waive rights to privacy and compensation, which I may have
+                    in connection with such use of my name and likeness,
+                    inlcuding rights to be written copy that may be created in
+                    connection with video production, editing, and promotion
+                    therewith.
+                </Text>
+            </Box>
+            <Box>
+                <Text pb="5px" align="left" mt="30px">
+                    If under the age of 19:
+                </Text>
+                <Text pb="5px" align="left" mt="30px">
+                    I hereby authorize any images or video footage taken of my
+                    youth (under 18 years of age), in whole or in part,
+                    individually or in conjunction with other image and video
+                    footage, to be displayed on the Social Diversity for
+                    Children Foundation and other official channels by Social
+                    Diversity for Children Foundation or its partners, sponsors,
+                    or affiliated entities, and to be used for media purposes
+                    including promotional presentations, marketing campaigns,
+                    paper media, broadcast media, brochures, pamphlets,
+                    materials, books, and all other avenues. I also authorize
+                    the display and use of any media material created by my
+                    youth within the Social Diversity for Children Foundaiton.
+                </Text>
+                <Text pb="60px" align="left" mt="30px">
+                    I waive rights to privacy and compensation, which I may have
+                    in connection with such use of my youth's name and likeness,
+                    including rights to be written copy that may be created in
+                    connection with video production, editing, and promotion
+                    therewith. I am over 19 years-of-age and the parent or legal
+                    guardian of the youth, and I have read this waiver and am
+                    familiar with its content.
+                </Text>
+            </Box>
+            <Box>
+                <Checkbox
+                    mb="80px"
+                    onChange={() => setAcceptedTerms(!acceptedTerms)}
+                >
+                    I have read and agree to the terms above
+                </Checkbox>
+            </Box>
+            <Box pb="50px">
+                <Button
+                    height="50px"
+                    width="200px"
+                    borderRadius="6px"
+                    background={!acceptedTerms ? "#737373" : "#0C53A0"}
```

Comment:
Please import `colourTheme` to get our colour theme from our one source of truth. Otherwise try to stick with preset colours like "darkgray".

```suggestion
                    background={!acceptedTerms ? "darkgray" : colourTheme.colors.Blue}
```

Reply:
Note: we have 737373 as a colour them in kevin's branch. We will use that when we get the chance.